src/scripts/fix-program-events-user/config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "api": {
+        "url": "http://localhost:8080/api/24",
```

Comment:
@tokland My fault, sorry, but it was supposed to be targeting 2.25. Can you please update and test it for 2.25?